### PR TITLE
yay it works!

### PR DESCRIPTION
**Pull request by boyoonc**

### CODE REVIEW

File: src/ts/mmui-autocomplete/index.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/*
```
Comment (besseym) [inline src/ts/mmui-autocomplete/index.ts]: This looks good although I don't now if it needs to be in its own folder. There is a mmui-widget folder or we could create a mmui-component folder:
`mmu-widget/autocomplete.ts`
or 
`mmu-component/autocomplete.ts`

**Comment by boyoonc** [comment]
I think I'll also change the minlength and delay time as parameters with default values

File: src/ts/mmui-autocomplete/index.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+          terms.pop();
+          // add the selected item
+          terms.push( ui.item.value );
+          // add placeholder to get the comma-and-space at the end
+          terms.push( "" );
+          $(this).html(terms.join( ", " ));
+          this.value = terms.join( ", " );
+          return false;
+        },
+        // delay:200, //300 by default
```
Comment (xyz8983) [inline src/ts/mmui-autocomplete/index.ts]: nice chaining! good to know

**Comment by boyoonc** [comment]
please re-review. styles moved to mmui, and more parameters added. I'm not sure if we should allow the delimiters to be parameters

File: src/ts/mmui-widget/mmui-autocomplete.ts [hunk 3]
```diff
@@ -0,0 +1,129 @@
+/*
+you will need these imports in your ts, and the jquery-ui css in your scss to use these functions
+import * as $ from 'jquery';
+import 'jquery-ui';
+import * as popper from 'popper.js';
+import * as bootstrap from 'bootstrap';
+*/
+
+declare var $: any;
+export function singleAutocomplete(selector, parentSelector, url, minLength = 2, delay = 300){
```
[the remark this answers is not in the document]
Comment (boyoonc) [reply src/ts/mmui-widget/mmui-autocomplete.ts]: added!

File: src/ts/index.ts [hunk 4]
```diff
@@ -13,4 +13,4 @@ export {
     formObjToParameterUrl,
     objToParameterUrl
 } from "./mmui-web/index";
-export {PositionList} from "./mmui-widget/index";
+export * from "./mmui-widget/index";
```
Comment (besseym) [inline src/ts/index.ts]: Oh, didn't know this was possible.

**Pull request merged**